spec/models/outgoing_message_spec.rb

```diff
@@ -143,12 +143,45 @@
     expect(@outgoing_message.get_text_for_indexing).not_to include("foo@bar.com")
   end
 
-  it "should not display email addresses on page" do
-    expect(@outgoing_message.get_body_for_html_display).not_to include("foo@bar.com")
-  end
+  context "when formatting the message for html display" do
+
+    it "does not display email addresses on page" do
+      expect(@outgoing_message.get_body_for_html_display).not_to include("foo@bar.com")
+    end
+
+    it "links to help page where email address was" do
+      expect(@outgoing_message.get_body_for_html_display).to include('<a href="/help/officers#mobiles">')
+    end
+
+    it "does not force long lines to wrap" do
+      long_line = "long string of 125 characters, set so the old line break " \
+                  "falls here, and making sure even longer lines aren't " \
+                  "affected either"
+      @outgoing_message.body = long_line
+      expect(@outgoing_message.get_body_for_html_display).not_to include('<br>')
+    end
+
+    it "interprets single line breaks as <br> tags" do
+      split_line = "Hello,\nI am a test message\nWith multiple lines"
+      expected = "Hello,<br>I am a test message<br>With multiple lines"
+      @outgoing_message.body = split_line
+      expect(@outgoing_message.get_body_for_html_display).to include(expected)
+    end
+
+    it "interprets double line breaks as <p> tags" do
+      split_line = "Hello,\n\nI am a test message\n\nWith multiple lines"
+      expected = "Hello,<p>I am a test message<p>With multiple lines"
```

Comment:
I'd expect, at a minimum:

```
"Hello,<p>I am a test message</p><p>With multiple lines"
```

Shouldn't it really be:

```
"<p>Hello,</p><p>I am a test message</p><p>With multiple lines</p>"
```

I would not be adverse to directly calling [`simple_format`](to do this).